docs/RISKS.md

```diff
@@ -0,0 +1,40 @@
+| 5 | Technology failures | This may include breaking changes in our tech stack requiring immediate changes, as well as unexpected behavior of technology, ultimately leading to the result that the goal of the project is not achievable. | 0,1 | 10 | 1 | This risk can be best mitigated by two strategies: (a) choosing reliable technology with a record of stability as well as reliability (following best practices), (b) training team member to employ them with the power to response to technology failure immediately on their own behalf | Jonas |
+| 6 | Code quality / Technical debts | Technical debt refers to the shortcuts or compromises taken in the development process that can lead to problems down the road. This can include poorly written code, incomplete documentation, lack of testing, or outdated technology stacks. | 0,2 | 3 | 0,6 | This risk can be mitigated by establishing coding standards and best practices. By conducting code reviews, focus is taken on the possibility for further extension and growth. The team is reminded of the wish for refactoring whenever helpful, to adress technical debts early on and get rid of those. Tests can be used as a measure of confidence, so refactoring can be done easier because the behavior is tested. This includes unit tests, integration tests as well as regression tests. The team is aware that taking shortcuts is not wanted and will hinder us down the path when it'll become the most stressful. | Markus |
+| 7 | Hardware problems | Hardware may fail or malfunction for any reason, including theft. Firmware or human failures may result in a lock-out of our developers so that the hardware becomes unusable for our project. | 0,2 | 9 | 1,8 | This risk can be mitigated by regular maintenance and testing. By making sure all important work is pushed to GitHub, we have another layer of security in case someone looses access to their laptop. | Jens |
+| 8 | Team members get ill or don't pass exams | At this big complexity in our project, we are in demand of working capacity. If fellow students (team members) are not able to work because they are no longer part of the university due to failing exams or getting ill for a longer period of time, the project may not reach its objectives. | 0,2 | 10 | 2 | This risk is hard to mitigate. All of us need to make sure to pass exams and pay attention to both physical and mental health. Further, cross-training of team members should remain a focus to make sure we don't get into knowledge silos. We are providing additional support, for example tutoring und mentoring not only for this project but overall to make sure everyone stays with us. Planning for team member absences, if necessary, helps as well. | Chris |
```

Comment:
```suggestion
| 8 | Team members get ill or don't pass exams | At this big complexity in our project, we are in demand of working capacity. If fellow students (team members) are not able to work because they are no longer part of the university due to failing exams or getting ill for a longer period of time, the project may not reach its objectives. | 0,2 | 10 | 2 | This risk is hard to mitigate. All of us need to make sure to pass exams and pay attention to both physical and mental health. Further, cross-training of team members should remain a focus to make sure we don't get into knowledge silos. We are providing additional support, for example tutoring und mentoring not only for this project but overall to make sure everyone stays with us. Planning for team member absences, if necessary, helps as well. | Chris, Jonas |
```

I would also like to be part

Reply:
fine for me